### DISCUSSION TITLE
UserAgent is used for Playground MiniGames or other functions?

### DISCUSSION THREAD

**Discussion by AKTanara** [masterking32/MasterHamsterKombatBot#210] [General]
I tried to read and understand the code but couldn't. So I decided to ask.
Is UserAgent used for Playground MiniGames code obtaining functionality?
Thanks

**Comment by tboy1337** [comment]
A UserAgent is used but not the one you set in the config.

**Comment by AKTanara** [comment]
So is it the same userAgent for anyone using this code?
Isn't there any method to use unique userAgent to prevent banning?

**Reply by tboy1337** [reply]
> So is it the same userAgent for anyone using this code? Isn't there any method to use unique userAgent to prevent banning?

Its the same for everyone because the app you use to play the game is the same for everyone, if you had a unique one that would probably lead to you getting banned.

**Comment by Fy0urM** [comment]
the useragent you specified in config.py is used only for requests to the api of hamster kombat itself, for taps, buying cards, getting ciphers, etc. to get playground game keys for each game uses its own useragent obtained by analyzing the traffic of the game applications themselves and specified in promogames.py.
